module14/tests/test_hw14.py

```diff
@@ -0,0 +1,41 @@
+from module14.common.hw14 import DemoblizeHomePage
+from selenium import webdriver
+import pytest
+
+@pytest.fixture(scope='function')
+def driver():
+    _driver = webdriver.Chrome()
+    _driver.maximize_window()
+    yield _driver
+    _driver.quit()
+
+def test_login(driver):
+    test_page = DemoblizeHomePage(driver)
+    test_page.open_home_page()
+    test_page.open_log_in_form()
+
+    assert test_page.get_login().is_displayed(), 'Login field is not displayed'
+    assert test_page.get_password().is_displayed(), 'Password field is not displayed'
+
+    test_page.log_in()
+
+    assert test_page.get_logout_button().is_displayed(), 'Logout button is not displayed'
+    assert test_page.get_welcome_message().text == f'Welcome {test_page.user_login}'
+
+@pytest.fixture
+def login(driver):
```

Comment:
Fixtures are expected to be in conftest.py file (not so important in this case but generally it helps to maintain order in the project structure; content of conftest file is available for all test suites in its folder)

Reply:
Done